### PR TITLE
Change log for release-0.19.11

### PR DESCRIPTION
**Pull request by 2ez4szliu**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/whatsnew/changelog.md [hunk 1]
```diff
@@ -30,6 +30,34 @@ description: >
...
+## [v0.19.11](https://github.com/aws/eks-anywhere/releases/tag/v0.19.11)
+### Supported OS version details
+|                     | vSphere | Bare Metal | Nutanix | CloudStack | Snow |
+|:-------------------:|:-------:|:----------:|:-------:|:----------:|:----:|
+|    Ubuntu 20.04     |    ✔    |     ✔      |    ✔    |     —      |  ✔   |
+|    Ubuntu 22.04     |    ✔    |     ✔      |    ✔    |     —      |  —   |
+| Bottlerocket 1.22.0 |    ✔    |     \*      |    —    |     —      |  —   |
+|      RHEL 8.x       |    ✔    |     ✔      |    ✔    |     ✔      |  —   |
+|      RHEL 9.x       |    —    |     —      |    ✔    |     ✔      |  —   |
+
```
Comment (sp1999) [inline docs/content/en/docs/whatsnew/changelog.md:42]:
```suggestion

\* [EKS Anywhere issue regarding deprecation of Bottlerocket bare metal variants](https://github.com/aws/eks-anywhere/issues/7754)

```

File: docs/content/en/docs/whatsnew/changelog.md [hunk 2]
```diff
@@ -30,6 +30,34 @@ description: >
 * When upgrading to a new minor version, a new OS image must be created using the new image-builder CLI pertaining to that release.
 {{% /alert %}}
 
+## [v0.19.11](https://github.com/aws/eks-anywhere/releases/tag/v0.19.11)
+### Supported OS version details
+|                     | vSphere | Bare Metal | Nutanix | CloudStack | Snow |
+|:-------------------:|:-------:|:----------:|:-------:|:----------:|:----:|
+|    Ubuntu 20.04     |    ✔    |     ✔      |    ✔    |     —      |  ✔   |
+|    Ubuntu 22.04     |    ✔    |     ✔      |    ✔    |     —      |  —   |
+| Bottlerocket 1.22.0 |    ✔    |     \*      |    —    |     —      |  —   |
```
Comment (sp1999) [inline docs/content/en/docs/whatsnew/changelog.md:39]:
```suggestion
| Bottlerocket 1.20.5 |    ✔    |     \*      |    —    |     —      |  —   |
```

File: docs/content/en/docs/whatsnew/changelog.md [hunk 3]
```diff
@@ -24,12 +24,44 @@ description: >
   * [Building Ubuntu and Red Hat node images]({{< relref "../osmgmt/artifacts/#building-node-images" >}})
   * [Downloading Bottlerocket node images]({{< relref "../osmgmt/artifacts/#download-bottlerocket-node-images" >}})
   * [Upgrading an EKS Anywhere cluster]({{< relref "../clustermgmt/cluster-upgrades" >}})
+* On October 11, 2024, a security issue CVE-2024-9594 was discovered in the Kubernetes Image Builder where default credentials are enabled during the image build process when using the Nutanix, OVA, QEMU or raw providers. The credentials can be used to gain root access. Clusters using virtual machine images built with [Kubernetes Image Builder](https://github.com/kubernetes-sigs/image-builder) version `v0.1.37` or earlier are affected if built with the Nutanix, OVA, QEMU or raw providers. This CVE has been fixed in image-builder version `v0.1.39`, which has been included in EKS Anywhere release `v0.19.11`.
```
Comment (abhay-krishna) [inline docs/content/en/docs/whatsnew/changelog.md:27]:
```suggestion
* On October 11, 2024, a security issue CVE-2024-9594 was discovered in the Kubernetes Image Builder where default credentials are enabled during the image build process when using the Nutanix, OVA, QEMU or raw providers. The credentials can be used to gain root access. Clusters using virtual machine images built with [Kubernetes Image Builder](https://github.com/kubernetes-sigs/image-builder) version `v0.1.37` or earlier are affected if built with the Nutanix, OVA, QEMU or raw providers. This CVE has been fixed in image-builder versions `v0.1.38` and the fix is included in EKS Anywhere release `v0.19.11`.
```

Comment (2ez4szliu) [reply docs/content/en/docs/whatsnew/changelog.md]: Fixed. I think its `version >= 0.1.38`, right?

**Pull request merged**